src/backend/test.rs

```diff
@@ -102,6 +112,38 @@ impl TestBackend {
         crate::assert_buffer_eq!(&self.buffer, expected);
     }
 
+    /// Asserts that the `TestBackend`'s scrollback buffer is equal to the expected buffer.
+    ///
+    /// This is a shortcut for `assert_eq!(self.scrollback(), &expected)`.
+    ///
+    /// # Panics
+    /// When they are not equal, a panic occurs with a detailed error message showing the
+    /// differences between the expected and actual buffers.
+    #[allow(deprecated)]
+    #[track_caller]
+    pub fn assert_scrollback(&self, expected: &Buffer) {
+        // TODO: use assert_eq!()
+        crate::assert_buffer_eq!(&self.scrollback, expected);
+    }
+
+    /// Asserts that the `TestBackend`'s scrollback buffer is empty, but also has the given width.
+    ///
+    /// # Panics
+    /// When the scrollback buffer is not equal, a panic occurs with a detailed error message
+    /// showing the differences between the expected and actual buffers.
+    pub fn assert_empty_scrollback(&self, width: u16) {
```

Comment:
I don't find the arguments to this function intuitive. Maybe it would be leave testing this to the user instead of adding a opinionated function like this?

Reply:
The problem is that it's kind of a pain to specify an empty buffer using what's currently there. The assert is used in three places now, and I can imagine it being used in more in the future. Having to specify an empty buffer of the correct width at the assert-side is pretty clunky and verbose.

What about if I removed `width`? From the asserter's perspective, it doesn't really matter what the width is. It's really an implementation decision that the scrollback buffer starts out with any width at all. I think that may address your concern?

Reply:
Makes sense / looks good now! Thanks :)